libraries/AP_Proximity/AP_Proximity_Boundary_3D.cpp

```diff
@@ -178,6 +191,23 @@ void AP_Proximity_Boundary_3D::reset_face(const Face &face)
     update_boundary(face);
 }
 
+// check if a face has valid distance even if it was updated a long time back
+void AP_Proximity_Boundary_3D::check_face_timeout()
+{
+    for (uint8_t layer=0; layer < PROXIMITY_NUM_LAYERS; layer++) {
+        for (uint8_t sector=0; sector < PROXIMITY_NUM_SECTORS; sector++) {
+            if (_distance_valid[layer][sector]) {
+                const uint32_t dt =  AP_HAL::millis() - _last_update_ms[layer][sector];
```

Comment:
I would just remove this "dt" line.. I think it is just as clear to have the subtraction on the line below.